.github/workflows/send-redirects-to-staging.yml

```diff
@@ -28,9 +27,12 @@ jobs:
         run: |
           cat << EOF > terraform.auto.tfvars
             stage                 = "staging"
+            docs_access_key       = ${{ secrets.DOCS_AWS_KEY_ID }}
```

Comment:
@jandroav this won't expose the secrets in clear text in the logs will it?  Should we be doing these as vars in TF Cloud instead?

Reply:
@jnewton03 mmmmm I don't think so as referenced here https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow. We could store it in TF Cloud but we would need to remove the TF code created in the infra repo for storing the new IAM user keys for the docs team in GH (https://github.com/Datical/liquibase-infrastructure/pull/738/files). But we could make a dummy test in a dummy repo to check if that kind of secret is exposed.

Reply:
@jandroav lets move forward with this solution and if there's a problem we can solve it after.